easybuild/easyconfigs/b/build/build-0.10.0-foss-2022a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonBundle'
+
+name = 'build'
+version = '0.10.0'
+
+homepage = 'https://github.com/pypa/build'
+description = """A simple, correct Python build frontend."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('pyparsing', '3.0.9', {
+        'checksums': ['2b020ecf7d21b687f219b71ecad3631f644a47f01403fa1d1036b0c6416d70fb'],
+    }),
+    ('packaging', '20.4', {
```

Comment:
`pyparsing`, `packaging`, and `tomli` are already included with `Python`, so these extensions can be removed here probably?